notification/slack/channel.go

```diff
@@ -267,51 +269,66 @@ func (s *ChannelSender) loadChannels(ctx context.Context) ([]Channel, error) {
 	return channels, nil
 }
 
+// Send handles processing new alerts and message updates to be sent out to a Slack channel
+// Parameters & URL documented here:
+// https://api.slack.com/methods/chat.postMessage
 func (s *ChannelSender) Send(ctx context.Context, msg notification.Message) (*notification.MessageStatus, error) {
 	cfg := config.FromContext(ctx)
+	var a alert.Alert
+	var timestamps []string
 
-	vals := make(url.Values)
-	// Parameters & URL documented here:
-	// https://api.slack.com/methods/chat.postMessage
-	vals.Set("channel", msg.Destination().Value)
 	switch t := msg.(type) {
 	case notification.Alert:
-		vals.Set("text", fmt.Sprintf("Alert: %s\n\n<%s>", t.Summary, cfg.CallbackURL("/alerts/"+strconv.Itoa(t.AlertID))))
+		a.Summary = t.Summary
+		a.ID = t.AlertID
+		a.Status = alert.StatusTriggered
 	case notification.AlertBundle:
-		vals.Set("text", fmt.Sprintf("Service '%s' has %d unacknowledged alerts.\n\n<%s>", t.ServiceName, t.Count, cfg.CallbackURL("/services/"+t.ServiceID+"/alerts")))
+		a.Summary = fmt.Sprintf("Service '%s' has %d unacknowledged alerts.", t.ServiceName, t.Count)
```

Comment:
I think we should be good here since bundles should always come in as 1+n alerts, though this does remind me that we need to ensure we aren't filling in all the blocks for bundled messages since they won't work with bundles as is.